src/extension/cli/serverError.ts

```diff
@@ -0,0 +1,8 @@
+class ServerError extends Error {
+    constructor(message: string) {
+        super(message)
+        this.name = 'runme server error'
```

Comment:
```suggestion
        this.name = 'RunmeServerError'
```

JS error name convention